sunpy/visualization/tests/test_mapcubeanimator.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec  5 13:32:41 2017
+
+@author: ajit
+"""
+
+from __future__ import absolute_import
+
+import matplotlib.pyplot as plt
+
+import pytest
+
+import sunpy.map
+
+from sunpy.visualization import mapcubeanimator
+
+from sunpy.data import test
```

Comment:
this doesn't follow the correct grouping of the imports. Please, could you put all the sunpy ones without empty lines in between, and so matplotlib and pytest in its own group?